evalml/tests/utils_tests/test_dependencies.py

```diff
@@ -26,8 +29,11 @@ def test_has_minimal_deps(
     extra_deps += ["plotly.graph_objects"]
     for module in extra_deps:
         # We don't expect to install sktime in python 3.9. Let's verify it's not present:
-        if (module == "sktime" and is_running_py_39_or_above) or (
-            module == "pmdarima" and is_using_conda
+        if (
+            (module == "sktime" and is_running_py_39_or_above)
+            or (module == "pmdarima" and is_using_conda)
+            or (module == "cmdstanpy" and (is_using_windows or is_using_conda))
```

Comment:
Do we need the extra cases for "cmdstanpy" and "prophet"? Those are not listed in the requirements.txt right?

Reply:
Oops these were leftover from when they were in requirements